### ISSUE TITLE
Copy Contact Address from Accounts Ship To Address

### ISSUE THREAD

**Issue by jobvector** [salesagility/SuiteCRM#885]
In the process of creating a Contact from the Accounts Subpanel Contacts the Contact address data is copied from the "Bill To Address" of the Account.

A solution where the Contact address can be copied from Accounts "Ship To Address" would be nice.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Copy Contact Address from Accounts Ship To Address](https://trello.com/c/XFRLozUB/120-copy-contact-address-from-accounts-ship-to-address)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
